crates/revm/src/db/states/bundle_account.rs

```diff
@@ -154,7 +154,7 @@ impl BundleAccount {
             AccountInfoRevert::DoNothing
         };
 
-        match updated_status {
+        let account_revert = match updated_status {
```

Comment:
There is one branch that uses `return`  (like 251) we can just use `else` there to cover all of them

Reply:
> There is one branch that uses `return` (like 251) we can just use `else` there to cover all of them

Reply:
Ok I've removed every early return inside that function (there were two). It should be easier to reason about it now